config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -43,6 +43,10 @@ presubmits:
               memory: "9000Mi"
               # during the tests more like 3-20m is used
               cpu: 2000m
+          env:
+            # This E2E_VERBOSITY 8 is for outputting API operations on e2e.log
+            - name: E2E_VERBOSITY
```

Comment:
So I'm not sold we want this verbosity here.  That is a lot of noise.

Reply:
Can we get a run with the increased verbosity to consider:
 1) what information you needed that was added
 2) confirm that the information is/isn't available in another place already
 3) confirm whether or not `8` was too much noise